### ISSUE TITLE
GCAMs compatibility

### ISSUE THREAD

**Issue by alibaubau** [deckerst/aves#781]
**Is your feature request related to a problem? Please describe.**
Aves v1.9.3 is a wonderful app, but viewing photos inside a Gcam app is only occasionally operating well: 
when pushing the viewer button the display blinks sometimes, seldom shows the photo and most of the time displays a blank screen.
This behavior is the same if the photo was shot immediately before or not (processing time can be very long).

**Describe the solution you'd like**
Adapt the program to work with Gcams.

**Describe alternatives you've considered**
Google photo and other gallery apps (i.e. Simple Gallery) operate well in Gcam, but I prefer to stay with Aves for its outstanding features and continue to exit Gcam, open Aves and back.

**Additional context**
I tried different Gcams and versions (I am using LMC v8.4) but all behave the same way.
Problem was already present in previous phone original ROM (Xiaomi A13), now it is LineageOS v20, rooted.
I have also adb-uninstalled the default gallery without success.

**Comment by deckerst** [comment]
Is Gcam an app I can install on my regular phone? From what you say it seems there are many variants. Which Gcam would you recommend? Do you have an APK/store for that variant?

**Comment by alibaubau** [comment]
> Is Gcam an app I can install on my regular phone? From what you say it seems there are many variants. Which Gcam would you recommend? Do you have an APK/store for that variant?

Yes, you can install a Gcam in almost all phones in parallel with the phone default camera. 
A Gcam is a Pixel Google Camera app modded plus eventually a config file to adapt to each specific phone; the 8.4 versions are popular while, imho, newer versions are not yet fully operative. NB: Google services (or substitute) are necessary.
There are many modders and a site regroups a lot of them at www.celsoazevedo.com, while the FAQ at www.celsoazevedo.com/files/android/google-camera/troubleshooting/ is a great help to start. There are also many Telegram groups.
I myself use an LMC mod from Hasli plus an Arcide config for Mi11Ultra. Photos' quality is better than with the original Xiaomi camera.

**Comment by deckerst** [comment]
Just tried `LMC8.8_beta_fix (Hasli, 2023-09-27)` on my Samsung S10e (Android 12, API 31): the app photo mode does not work but the video and panorama ones do, and opening them with Aves works fine.

Could you please attach here the environment info and logs you get from Aves > About > Bug Report? If you know about logcat, please attach logcat output when you try to open Aves and it fails.

Also, maybe I misunderstood your report so could you please make a screen recording of the issue?

**Comment by deckerst** [comment]
I also tried `LMC8.4_R18F1`, which is more stable, with the Camera mode working as expected. Again, opening the photos with Aves was working fine.

The only exception is when I tried to take many pictures in rapid fire and quickly open the last one, it opened a pending photo (which is incomplete, still being processed by the camera, its name starting with ".pending-{timestamp}-_PXL"). In that case, the placeholder image is not decodable by Aves, that's normal. Coming back to GCam and reopening the viewer with Aves worked fine, as the final photo was processed and complete.
